source/vibe/web/common.d

```diff
@@ -319,51 +337,29 @@ PathAttribute path(string data)
 
 
 /**
-	UDA to define root URL prefix for annotated REST interface.
-	Empty path means deducing prefix from interface type name (see also rootPathFromName)
+	Deprecated. Use @$(D path) instead.
  */
-RootPathAttribute rootPath(string path)
+deprecated(`Use @path("path") directly.`)
```

Comment:
I misread your comment (https://github.com/rejectedsoftware/vibe.d/pull/999#issuecomment-86201613), and thought you were okay with deprecating `@rootPath` but not `@rootPathFromName`.
So this will become nothing but a documentation commit. If I had knew, I would have pushed to get this into 0.7.23 ;)